docs/dev/ui_style_guide.rst

```diff
@@ -0,0 +1,39 @@
+***************************
+UI/UX Style Guide
+***************************
+
+Tray Plugins
+-------------
+
+In order to be consistent with layout, styling, and spacing, UI development on plugins should
+try to adhere to the following principles.
+
+* any tray plugin should utilize ``<j-tray-plugin>`` as the outer-container (which provides consistent 
+  styling rules).  Any changes to style across all plugins should then take place in the 
+  ``j-tray-plugin`` stylesheet (``tray_plugin.vue``).
+* each item should be wrapped in a ``v-row``, but avoid any unnecessary additional wrapping-components
+  (``v-card-*``, ``v-container``, etc).
+* only use ``v-col`` components (within the ``<v-row class="row-no-outside-padding">``) if multiple 
+  components are necessary in a single row.  Always emphasize readability at the default/minimum
+  width of the plugin tray, rather than using columns that result in a ton of text overflow.
+* action buttons should have ``color="primary"`` if it loads something into the plugin, or 
```

Comment:
Jenn and I talked about this a bit too and decided to *try* to adhere to this pattern and see if we get any feedback.  Ultimately it only does harm if it adds confusion, but if it just often goes unnoticed, then nothing is really lost.  If anyone finds this significantly confusing, it can easily be reverted to all be the same.  The most obvious case is in the line list plugin where "load lines" is blue and "plot/erase all" are orange.